package.json

```diff
@@ -5,8 +5,8 @@
   },
   "preferGlobal": false,
   "dependencies": {
-    "@qooxdoo/compiler": "^1.0.0-beta",
-    "@qooxdoo/eslint-config-qx": "^1.2.1",
-    "@qooxdoo/eslint-plugin-qx": "^1.2.1"
+    "@qooxdoo/framework": "^7.3.0",
```

Comment:
do we actually REQUIRE this

Reply:
for building with npx

Reply:
Or do not use npxin the docker?

Reply:
the reason I am asking is, because I am wondering if this will cause problems for users who have not yet updated to qx7